cds/modules/theme/assets/bootstrap3/scss/cds_previewer/video.scss

```diff
@@ -65,3 +65,31 @@ video {
 .theo-primary-background {
   background-color: #d9d9d9 !important;
 }
+
+// LIGHT SKIN FOR EMBED PLAY BUTTON
+.theo-light-skin.theoplayer-skin .vjs-big-play-button{
```

Comment:
```suggestion
theoplayer-skin.light .vjs-big-play-button{
```

Can we just name it `light` as this is applied already in the `theoplayer-skin` class?